src/scenes.jl

```diff
@@ -454,7 +454,8 @@ function Base.push!(plot::Plot, subplot)
     push!(plot.plots, subplot)
```

Comment:
We could add the check here as well, to ensure full coverage and so that e.g. attributes passed through recipes are caught.

```suggestion
    MakieCore.validate_attribute_keys(Plot{Typ}, plot.kw)
    push!(plot.plots, subplot)
```